src/engine/renderer/tr_init.cpp

```diff
@@ -1107,7 +1107,14 @@ ScreenshotCmd screenshotPNGRegistration("screenshotPNG", ssFormat_t::SSF_PNG, "p
 		r_subdivisions = Cvar_Get( "r_subdivisions", "4", CVAR_LATCH );
 		r_dynamicLightCastShadows = Cvar_Get( "r_dynamicLightCastShadows", "1", 0 );
 		r_precomputedLighting = Cvar_Get( "r_precomputedLighting", "1", CVAR_CHEAT | CVAR_LATCH );
-		r_vertexLighting = Cvar_Get( "r_vertexLighting", "0", CVAR_LATCH | CVAR_ARCHIVE );
+		/* Default lighting mode, the engine fallbacks on lower ones if missing for the
+		whole map or for the given surface.
+		This is a cheat cvar as changing the lighting mode may give an unfair advantage.
+		0: fullbright
+		1: vertex lighting
+		2: grid lighting
+		3: light mapping */
+		r_lighting = Cvar_Get( "r_lighting", "3", CVAR_LATCH | CVAR_CHEAT );
```

Comment:
do the Cvar::Range

Reply:
We need this to make it lateched:

- https://github.com/DaemonEngine/Daemon/pull/999